.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+AllCops:
+  TargetRubyVersion: 2.4
+
+Metrics/LineLength:
+  Max: 80
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false # Using Ruby 2.4 doesn't need it
```

Comment:
I think its actually the opposite, and we need this enabled, adding this comment makes ruby 2.3+ behave like ruby 3.0 will (potentially) in that all string will be immutable